doc/source/io.rst

```diff
@@ -3965,72 +3982,173 @@ function.
-   gbq.generate_bq_schema(df, default_type='STRING')
+    df.to_gbq('my_dataset.my_table', projectid, if_exists='fail')
 
-.. warning::
+.. note::
+
+    If the ``if_exists`` argument is set to ``append``, the destination dataframe will
+    be written to the table using the defined table schema and column types. The
+    dataframe must match the destination table in column order, structure, and
+    data types.
+    If the ``if_exists`` argument is set to ``replace``, and the existing table has a
+    different schema, a delay of 2 minutes will be forced to ensure that the new schema
+    has propagated in the Google environment. See
+    `Google BigQuery issue 191 <https://code.google.com/p/google-bigquery/issues/detail?id=191>`__.
+
+Writing large DataFrames can result in errors due to size limitations being exceeded.
+This can be avoided by setting the ``chunksize`` argument when calling :func:`~pandas.DataFrame.to_gbq`.
+For example, the following writes ``df`` to a BigQuery table in batches of 10000 rows at a time:
+
+.. code-block:: python
+
+    df.to_gbq('my_dataset.my_table', projectid, chunksize=10000)
+
+You can also see the progress of your post via the ``verbose`` flag which defaults to ``True``.
+For example:
+
+.. code-block:: python
+
+    df.to_gbq('my_dataset.my_table', projectid, chunksize=10000, verbose=True)
+
+.. note::
+
+    If an error occurs while streaming data to BigQuery, see
+    `Troubleshooting BigQuery Errors <https://cloud.google.com/bigquery/troubleshooting-errors>`__.
+
+.. note::
+
+    The BigQuery SQL query language has some oddities, see the
+    `BigQuery Query Reference Documentation <https://developers.google.com/bigquery/query-reference>`__.
+
+.. note::
+
+    While BigQuery uses SQL-like syntax, it has some important differences from traditional
+    databases both in functionality, API limitations (size and quantity of queries or uploads),
+    and how Google charges for use of the service. You should refer to `Google BigQuery documentation <https://developers.google.com/bigquery/>`__
+    often as the service seems to be changing and evolving. BiqQuery is best for analyzing large
+    sets of data quickly, but it is not a direct replacement for a transactional database.
+
+
+Creating BigQuery Tables
+''''''''''''''''''''''''
+
+As of 0.17.0, the gbq module has a function :func:`~pandas.io.gbq.create_table` which allows users
+to create a table in BigQuery. The only requirement is that the dataset must already exist.
+The schema may be generated from a pandas DataFrame using the :func:`~pandas.io.gbq.generate_bq_schema` function below.
+
+For example:
+
+.. code-block:: python
+
+    gbq.create_table('my_dataset.my_table', schema, projectid)
+
+As of 0.15.2, the gbq module has a function :func:`~pandas.io.gbq.generate_bq_schema` which will
+produce the dictionary representation schema of the specified pandas DataFrame.
+
+.. code-block:: python
+
+    df = pd.DataFrame({'my_string': list('abc'),
```

Comment:
you don't need this as you have it above (you can certainly put in a reference if you want to jump to it though)